fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -1330,11 +1330,11 @@ private PutLoansLoanIdChanges() {}
             @Schema
             public String dueDate;
             @Schema(example = "1")
-            public Integer chargeTimeType;
+            public Long chargeTimeType;
```

Comment:
This looks wrong to me based on it is the ordinal of an enum.

Reply:
Done.